src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -974,6 +974,7 @@ def cloud_init_user_data_func(
             protocol="tcp",
             security_groups=[
                 edxapp_security_group.id,
+                # operations_vpc["security_groups"]["celery_monitoring"],  # noqa: ERA001
```

Comment:
This line was causing the stack to not run. Will need to revisit and figure out why.

Reply:
That's because I had changed the structure of the export to have a `name` and `id` attribute, but that didn't work anyway because apparently cross-vpc security groups assigned to a pod don't work